MODULE.bazel.lock

```diff
@@ -9071,7 +9071,7 @@
     },
     "@@rules_python~//python/extensions:pip.bzl%pip": {
       "os:osx,arch:aarch64": {
-        "bzlTransitiveDigest": "E2QiQlkDaM1FJ0uHgq2yD7CHDc3lTZRZe6qqYlEYiRI=",
```

Comment:
I actually don't think this should be updated, can you please revert this ?

Reply:
Can you please do a `bazel clean --expunge`, it might because you were hitting https://github.com/bazel-contrib/bazel_features/issues/25